posthog/models/feedback/survey.py

```diff
@@ -40,6 +40,14 @@ class Meta:
         related_name="surveys_targeting_flag",
         related_query_name="survey_targeting_flag",
     )
+    custom_targeting_flag: models.ForeignKey = models.ForeignKey(
```

Comment:
naming nit, can we call this `internal_targeting_flag` instead? Or something else that better represents usage of this flag for dismiss / repeating surveys

Reply:
sounds good to me! I was debating between `system_targeting_flag` or `private_targeting_flag`, `internal_targeting_flag` sounds good to me.